bootstrap/kubeadm/internal/ignition/clc/clc.go

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
Commenting here given that this is the first file I see in the new `ignition` package. 

We should have an OWNERS file in here with 1-2 maintainers and reviewers, given that personally I don't know how ignition works, we should have some folks assigned to issues or PRs related to it

@CecileRobertMichon thoughts?

Reply:
+1, it totally agree
This is a really specific domain, and having in the loop people who can properly maintain this code will be super helpful.
It also helps to spread more evenly the review workload, which is definitely good

Reply:
+1

Reply:
Sure. @dongsupark, @iaguis can I put you here together with myself?

Reply:
Sure.

Reply:
> The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as members of the kubernetes-sigs org. You can then trigger verification by writing /verify-owners in a comment.

> dongsupark
User is not a member of the org. Satisfy at least one of these conditions to make the user trusted.
iaguis
User is not a member of the org. Satisfy at least one of these conditions to make the user trusted.
invidian
User is not a member of the org. Satisfy at least one of these conditions to make the user trusted.

hmm... :smile:

Reply:
Happy to sponsor folk if you file a request at https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

Reply:
I'm happy to apply, I have 13 contributions for `kubernetes-sigs` and `kubernetes` orgs, but I'd need to find another sponsor I guess :smile:

Reply:
@dongsupark has even more. Maybe @rata could sponsor us too? :smile:

Reply:
Happy to sponsor as well

Reply:
Sure, I'm part of kubernetes github org and willing to help :)

Reply:
I've created https://github.com/kubernetes/org/issues/2622 then. I hope I didn't rush too much with it :crossed_fingers:

Reply:
@dongsupark and I are now part of `kubernetes-sigs` and we are added as reviewers. Is there anything else what needs to be done?

Reply:
Marking as resolved since now we have owners.